tests/upload-helper.int.test.ts

```diff
@@ -32,6 +32,7 @@ describe('Integration Upload Helper', function() {
       testBucket,
       './tests/testdata/test1.txt',
       true,
+      true,
```

Comment:
Should we have a test with resumable = false?

Reply:
Done